.github/RELEASE.md

```diff
@@ -0,0 +1,265 @@
+<!-- These docs are updated and synced from https://github.com/conda/infra -->
+
+## Release Process
+
+[epic template]: issues/new?assignees=&labels=epic&template=epic.yml
+[rever docs]: https://regro.github.io/rever-docs
+[compare]: compare
+[release notes]: releases/new
```

Comment:
Are these two links currently just placeholders?

Reply:
**but** after further reflection I believe the relative URL will still be wrong, when we view this file on GH it'll be at, e.g., https://github.com/conda/conda/blob/main/RELEASE.md so we want:

```suggestion
[compare]: ../../compare
[release notes]: ../../releases/new
```